lib/src/benchmark_base.dart

```diff
@@ -1,64 +1,58 @@
 // Copyright 2011 Google Inc. All Rights Reserved.
 
-part of benchmark_harness;
+import 'score_emitter.dart' show PrintEmitter, ScoreEmitter;
 
 class BenchmarkBase {
   final String name;
   final ScoreEmitter emitter;
 
-  // Empty constructor.
   const BenchmarkBase(this.name, {this.emitter = const PrintEmitter()});
 
-  // The benchmark code.
-  // This function is not used, if both [warmup] and [exercise] are overwritten.
+  /// The benchmark code.
+  /// This function is not used, if both [warmup] and [exercise] are
+  /// overwritten.
   void run() {}
 
-  // Runs a short version of the benchmark. By default invokes [run] once.
-  void warmup() {
-    run();
-  }
+  /// Runs a short version of the benchmark. By default invokes [run] once.
+  void warmup() => run();
 
-  // Exercices the benchmark. By default invokes [run] 10 times.
+  /// Exercices the benchmark. By default invokes [run] 10 times.
   void exercise() {
     for (var i = 0; i < 10; i++) {
       run();
     }
   }
 
-  // Not measured setup code executed prior to the benchmark runs.
+  /// **Not measured** setup code executed prior to the benchmark runs.
   void setup() {}
 
-  // Not measures teardown code executed after the benchark runs.
+  /// **Not measures** teardown code executed after the benchark runs.
   void teardown() {}
 
-  // Measures the score for this benchmark by executing it repeately until
-  // time minimum has been reached.
-  static double measureFor(Function f, int minimumMillis) {
-    var minimumMicros = minimumMillis * 1000;
-    var iter = 0;
-    var watch = Stopwatch();
+  /// Measures the score for this benchmark by executing it repeately until
+  /// time minimum has been reached.
+  static double measureFor(Function functionToBeMeasured, int minimumMillis) {
```

Comment:
I think "benchmark" or "benchmarkFunction" is as clear as "functionToBeMeasured".
Full words are preferred over abbreviations, so I think minimumMilliseconds is better than minimumMillis